pydantic/main.py

```diff
@@ -266,14 +267,21 @@ class BaseModel(metaclass=ModelMetaclass):
 
     def __init__(__pydantic_self__, **data: Any) -> None:
         # Uses something other than `self` the first arg to allow "self" as a settable attribute
+        __pydantic_self__.__class__.__from_dict__(data, __pydantic_self__)
```

Comment:
```suggestion
        __pydantic_self__.__from_dict__(data, __pydantic_self__)
```

if we do it this way.